packages/web-runtime/src/store/user.js

```diff
@@ -98,12 +101,44 @@ const actions = {
 
         const userGroups = await client.users.getUserGroups(login.id)
         const user = await client.users.getUser(login.id)
+        const {
```

Comment:
needs a capability check as well. API is not available in oc10.

Reply:
There is none. Use the `spaces.enabled` capability for now, like it's already done for loading the user. We need to come up with better fitting capabilities though.